src/analysis/calculus/specific_functions.lean

```diff
@@ -187,3 +194,155 @@ begin
 end
 
 end exp_neg_inv_glue
+
+/-- An infinitely smooth function `f : ℝ → ℝ` such that `f x = 0` for `x ≤ 0`,
```

Comment:
Is this function monotone? I assume it is, but I don't see a lemma below stating that fact

Reply:
It is but this property is not required to build a partition of unity, and I'm too lazy to prove it without a good reason.